src/migrateCalculatedMeasures.ts

```diff
@@ -40,8 +39,7 @@ export async function migrateCalculatedMeasures(
 
     const id = generateId();
 
-    // @ts-ignore `children` will always be defined since it's hardcoded as an empty object above.
-    contentFolder.children[id] = {
+    contentFolder.children![id] = {
```

Comment:
In my suggestion was still keeping a comment to explain why casting as non-nullable is safe.
Can you add the comment back ?
And in general, if you agree with a GitHub suggestion, could you use the "Commit suggestion" feature ? It makes both the developer and the reviewer gain time.

Reply:
Yes definitely! In this case, I don't see the option to commit your suggestion directly so I'll a new commit manually.